source/PlayerInfo.h

```diff
@@ -431,7 +433,7 @@ class PlayerInfo {
 	// Persons that have been killed in this player's universe:
 	std::vector<std::string> destroyedPersons;
 	// Events that are going to happen some time in the future:
-	std::list<GameEvent> gameEvents;
+	std::multiset<GameEvent> gameEvents;
```

Comment:
I dont understand why this was changed?

Reply:
~~Yes, in the PR description~~

Reply:
~~could be good to have it in the code, mb~~